BGAnimations/ScreenSelectMusic overlay/playerModifiers.lua

```diff
@@ -20,27 +20,3 @@ for player in ivalues(GAMESTATE:GetHumanPlayers()) do
 		end
 	end
 end
-
--- ---------------------------------------------
```

Comment:
I'm pretty sure this code is obsolete. Applying mods for players late joining on SSM seems to work perfectly fine without it.